### ISSUE TITLE
Hue Bridge: warning: not using recommended Hue bridge api version 1.60.0

### ISSUE THREAD

**Issue by ruudbol** [ebaauw/homebridge-hue#1177]
I get the following message in the Homebridge log

Hue Bridge: warning: not using recommended Hue bridge api version 1.60.0

**Comment by ebaauw** [comment]
Looks like Signify upped the API version, as usual without any change log.  Does Homebridge Hue otherwise work as expected?

**Comment by sepp4729** [comment]
It doesn't work as expected. I was wondering why i got that error at first, after wanting to add a new Hue Bridge to Homebridge. I created a new Proxmox Container with Homebridge just for that Hue Bridge to test but it's stuck at "Starting..." and the error "[homebridge-hue] This plugin is taking long time to load and preventing Homebridge from starting. See https://homebridge.io/w/JtMGR for more info." Also a lot of errors with "timeout after 5 seconds - retrying in 15s". The other Hue Bridges are still working fine, but probably just a matter of time before they decide to update to the new API Version. (On the existing Homebridge it was also stuck on "starting..." Bridge. Hope there's a fix for it before Hue Updates all bridges. Maybe i'll cut their internet for the moment. 

But wait:
Software of "not working Bridge": 1.61.1961076030. Existing Bridge that is still working as expected: 1.61.1961076030. Now that i looked that up i'm confused.

**Comment by permaMonk** [comment]
Suddenly I´ve got there log messages:
[Hue] Philips Bridge 1: Signify Netherlands B.V. BSB002 bridge v1962097030, api v1.62.0
[Hue] Philips Bridge 1: warning: not using recommended Hue bridge api version 1.61.0
[Hue] Philips Bridge 1: request 1: /: api error 1: unauthorized user
[Hue] Philips Bridge 1: /: api error 1: unauthorized user - retrying in 15s
[Hue] Philips Bridge 1: Signify Netherlands B.V. BSB002 bridge v1962097030, api v1.62.0
[Hue] Philips Bridge 1: warning: not using recommended Hue bridge api version 1.61.0
[Hue] Philips Bridge 1: request 1: /: api error 1: unauthorized user
[Hue] Philips Bridge 1: /: api error 1: unauthorized user - retrying in 15s
[homebridge-hue] This plugin is taking long time to load and preventing Homebridge from starting. See https://homebridge.io/w/JtMGR for more info.

Same reason as above? What could I do?

**Comment by ebaauw** [comment]
The "not using recommended Hue bridge api version 1.61.0" warning is the same as above.  The "api error 1: unauthorized user" error is because an invalid (or no longer valid?) username is specified in config.json.  Best delete the entry from config.json, restart Homebridge and press the Hue bridge button to create a new username.  Make sure to copy/paste the bridgeid and username from the log (instead of manually retyping); they're case sensitive.

**Comment by permaMonk** [comment]
That's exactly what I did. I deleted the entry of the physical Hue Bridge and then the log was quiet.

But the question already exists for me. Why is the user, that has been running and valid for years, no longer valid? Did this happen through the API update?

I've already done this dozens of times by restarting the Homebridge and pressing the Hue Bridge button shortly afterwards. But neither the Homebridge user and password nor any other Hue Bridge entries appear in the log file.

**Comment by ebaauw** [comment]
> Why is the user, that has been running and valid for years, no longer valid? Did this happen through the API update?

You’ll have to ask the Hue bridge.  I’ve never seen before that a Hue bridge firmware update invalidates usernames.  A factory reset of the Hue bridge would.

> But neither the Homebridge user and password nor any other Hue Bridge entries appear in the log file.

They’re logged only after a new username has been obtained.  Once that’s copied correctly to config.json, it will be used.  In that case pressing the button on the Hue bridge won’t do anything.  Only when there’s no username in config.json, you’ll see a message in the log asking to press the Hue bridge button.

**Comment by permaMonk** [comment]
So, I've come to use the things you suggested.I deleted the two user entries. I added the physical Hue Bridge as an IP address under hosts. Restart. Then the log asks you to press the Hue Bridge button. Pressed. The user and the new password were then displayed in the log. Result - everything is fine.In addition, the error message that the correct API was not available (1.62 instead of 1.61) has also disappeared. To the last point - logic is different.—Am 13.01.2024 um 11:35 schrieb Erik Baauw ***@***.***>:﻿

Why is the user, that has been running and valid for years, no longer valid? Did this happen through the API update?

You’ll have to ask the Hue bridge.  I’ve never seen before that a Hue bridge firmware update invalidates usernames.  A factory reset of the Hue bridge would.

But neither the Homebridge user and password nor any other Hue Bridge entries appear in the log file.

They’re logged only after a new username has been obtained.  Once that’s copied correctly to config.json, it will be used.  In that case pressing the button on the Hue bridge won’t do anything.  Only when there’s no username in config.json, you’ll see a message in the log asking to press the Hue bridge button.
